bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -46,6 +46,7 @@ export default {
     { fields: { presenter: 1 } }).presenter,
   amIModerator: () => Users.findOne({ userId: Auth.userID },
     { fields: { role: 1 } }).role === ROLE_MODERATOR,
+  getPresenter: () => Users.findOne({ presenter: true }),
```

Comment:
Should you not pass meetingId?

Reply:
Changed the query by including meetingID and online status